src/components/doc-explorer/TypeList.tsx

```diff
@@ -37,15 +37,15 @@ export default class TypeList extends React.Component<TypeListProps> {
     );
 
     function renderItem(type, className?: string) {
-      if (!isMatch(type.name, filter)) {
+      if (!(isMatch(type.name, filter) || isMatch(type.description, filter))) {
         return null;
       }
 
       return (
         <div key={type.id} className={classNames('typelist-item', className)}>
           <TypeLink type={type} onClick={onTypeLink} filter={filter} />
           <FocusTypeButton onClick={() => onFocusType(type)} />
-          <Description className="-doc-type" text={type.description} />
+          <Description className="-doc-type" text={type.description} termToHighlight={filter} />
```

Comment:
Please use a consistent naming either rename it to `filter` or create separate PR that renames `filter` to something else in current codebase.

Reply:
I've used the `termToHighlight` as a prop name for `<Markdown>` and `<Description>` (that wraps `<Markdown>`) components, because *inside* these components it's the only meaningful name (generally speaking, the value that should be highlighted doesn't have to be a search term). Outside of these components I've preserved the `filter` and `searchValues` that are not consistent throughout the codebase (in each case it depends of the context).